client/src/modules/stock/entry/modals/findTransfer.modal.js

```diff
@@ -20,26 +22,34 @@ function StockFindTransferModalController(Instance, StockService, Notify,
   filtering = new Filtering(vm.gridOptions);
 
   columns = [
-    { field : 'status',
+    {
+      field : 'status',
       displayName : 'FORM.LABELS.STATUS',
       headerCellFilter : 'translate',
-      cellTemplate : 'modules/stock/entry/modals/templates/transfer.status.tmpl.html' },
+      cellTemplate : 'modules/stock/entry/modals/templates/transfer.status.tmpl.html',
+    },
 
-    { field : 'date',
+    {
+      field : 'date',
       cellFilter : 'date',
```

Comment:
Note: we should add the `bhConstants` date format parameter here to ensure it renders the same across French/English clients.